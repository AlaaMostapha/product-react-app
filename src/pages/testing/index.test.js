import React from "react";
import renderer from "react-test-renderer";
import Show from "./show";
it("renders no item", () => {
  const tree = renderer.create(<Show />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders 1 item", () => {
  const tree = renderer.create(<Show items={["a"]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
