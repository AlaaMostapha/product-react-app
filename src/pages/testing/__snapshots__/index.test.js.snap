// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1 item 1`] = `
<div>
  <div>
    a
  </div>
</div>
`;

exports[`renders no item 1`] = `<div />`;
